contrib/resources/glshaders/misc/fixvideo-scanlines.glsl

```diff
@@ -0,0 +1,171 @@
+#version 120
+
+/*
+ *  Adaptation of the old fixvideo.fx Daum shader originally written by Ove
+ *  Kavven with some improvements and an intelligent scanline-doubling method
+ *  added by John Novak.
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *
+ *  Copyright (C) 2023 The DOSBox Team
```

Comment:
Suggest clarifying and attributing the teams on separate lines (as many as needed). The DOSBox Team will have merges from: https://www.dosbox.com/crew.php

Reply:
My mistake, I should've put DOSBox Staging Team there. Fixed.